docs/website/sidebars.json

```diff
@@ -95,8 +95,5 @@
       "releases/v0-10-1",
       "releases/v0-10-0"
     ]
-  },
-  "": {
```

Comment:
Do not we have FAQ anymore?

Reply:
The FAQs page doesn't have a sidebar, so this is not needed.